### ISSUE TITLE
Gravity values are saved when switching to and from Planet X

### ISSUE THREAD

**Issue by phet-steele** [phetsims/pendulum-lab#86]
1. On the Lab screen, adjust the gravity to any desired value.
2. Switch to "Planet X" gravity.
3. Switch to  Moon, Earth, or Jupiter (Custom is not as dramatic).
4. Your value from step 1 will still be displayed, incorrectly, as that planet's gravity.

https://drive.google.com/file/d/0B3HJopSo_QqLVEgwc3VqT1hESzQ/view?usp=sharing

Tested on Win 8.1 Chrome

Troubleshooting information:
Name: Pendulum Lab
URL: http://www.colorado.edu/physics/phet/dev/html/pendulum-lab/1.0.0-dev.4/pendulum-lab_en.html
Version: 1.0.0-dev.4 2015-06-04 17:46:07 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.81 Safari/537.36
Language: en-US
Window: 1366x643
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 10 uniform: 253
Texture: size: 8192 imageUnits: 16 (vertex: 4, combined: 20)
Max viewport: 4096x4096
OES_texture_float: true
Dependencies JSON: {"comment":"# pendulum-lab 1.0.0-dev.4 Thu Jun 04 2015 11:46:00 GMT-0600 (MDT)","assert":{"sha":"903564a25a0f26acdbd334d3e104039864b405ef","branch":"master"},"axon":{"sha":"7de4a27be4b322b71c5f5046df193a03ccf0f161","branch":"master"},"babel":{"sha":"8725cd18114c1307057c465a0762a3cf8f1cb1b9","branch":"master"},"brand":{"sha":"b3362651f1d7eddb7f8697d02400472f97f9d140","branch":"master"},"chipper":{"sha":"c7d6ea5f9bef6e3882309af56ad409908dd28878","branch":"master"},"dot":{"sha":"a6a50c92c4cda5f8a3371d8c357ea565b7e7bd6c","branch":"master"},"joist":{"sha":"55fd6884039c5ef2f2145093c8530beab5490233","branch":"master"},"kite":{"sha":"b915f3b160744e4833d177dc38e74e4cbc912883","branch":"master"},"pendulum-lab":{"sha":"2b4cf70e4a8a88b37882b1643d44155a806508ab","branch":"master"},"phet-core":{"sha":"23834d107dc1cb368c00901c76927b712b9caa10","branch":"master"},"phetcommon":{"sha":"bb92c52cd90fdaa0c2a746fea82594afd439b4db","branch":"master"},"scenery":{"sha":"f09eddb7393946fcbedb63ebf6b88d1785c91bfc","branch":"master"},"scenery-phet":{"sha":"7ffc5a1a36e0afe36d035d0367078b3558fd157f","branch":"master"},"sherpa":{"sha":"ae2168a85ceb4094c23cd47ba5fe8145375448eb","branch":"master"},"sun":{"sha":"c9087fbb0cdd83f79452330fec3b83f809e2c0d8","branch":"master"}}

**Comment by arouinfar** [comment]
Also present in OSX 10.9.5/Chrome 43.

**Comment by arouinfar** [comment]
This issue exists on all screens, not just the Lab screen.

Desired behavior:
* If switching from Planet X to any of the non-Custom options (e.g. Moon), the slider/readout should match the value of that option (e.g. 1.62m/s^2).
* If switching from Planet X to Custom, the value should return the value of gravity selected immediately before Planet X was chosen. For example, if I select Jupiter, then Planet X, then Custom, the "Custom" value should match Jupiter (24.79m/s^2).  Alternatively, if I move the slider to a Custom value of 2/3 of the maximum, switch to Planet X, and then back to Custom, the slider should return to the same position.

**Comment by andrey-zelenkov** [comment]
Fixed gravity values saving. Reassign to @phet-steele for verification.

**Comment by arouinfar** [comment]
The previously described issues have been fixed. Thanks, @andrey-zelenkov!

However, I've noticed a new, though related issue. If the gravity is selected to be one of the benchmark values (e.g. Moon) and then the slider is moved, the display should change to "Custom." This currently behaves properly, but only for the <b>first</b> time the slider is used to adjust the gravity away from a benchmark value. One possible way to reproduce this issue:

1) Open the sim, select any screen, and displace a pendulum
2) Use the slider to change the gravity from Earth to some other value. The combo box will now show "Custom". (This is the desired behavior.)
3) Switch the gravity to a different benchmark, such a Jupiter.
4) Use the slider to change the gravity. The combo box still reads "Jupiter". It should read "Custom".

Reassigning to @andrey-zelenkov.
